readme.md

```diff
@@ -91,48 +94,32 @@ Default: `true`
 
 Whether to maintain image aspect ratio or not.
 
-#### imageBuffer
-
-Type: `Buffer`
-
-Buffer with the image.
-
-### terminalImage.file(filePath, options?)
-
-Returns a `Promise<string>` with the ansi escape codes to display the image.
-
-#### filePath
-
-Type: `string`
-
-File path to the image.
-
-##### options
-
-Type: `object`
+###### maximumFrameRate
 
-###### height
+**Only works for `terminalImage.gifBuffer` or `terminalImage.gifFile`**
 
-Type: `string | number`
+Type: `number`\
+Default: `Infinity`
 
-Custom image height.
+Maximum framerate to render the GIF.
```

Comment:
This should mention that in iTerm it ignores this setting and runs at full FPS as iTerm can render animated GIF natively.